scripts/ci/pylint_todo.txt

```diff
@@ -443,6 +443,7 @@
 ./airflow/sensors/time_sensor.py
 ./airflow/sensors/web_hdfs_sensor.py
 ./airflow/settings.py
+./airflow/sentry.py
```

Comment:
I think it would be preferable to fix the pylint warnings instead of adding the new files to the todo list. Please take this comment with a grain of salt though, I have been guilty of the same thing. 👼

Reply:
Fixed the Pylint errors